tests/Doctrine/Migrations/Tests/Tools/Console/Command/StatusCommandTest.php

```diff
@@ -73,6 +74,12 @@ public function testExecute(): void
 
         $lines = array_map('trim', explode("\n", trim($this->commandTester->getDisplay(true))));
 
+        if (DBALVersion::compare('2.11.0') > 0) {
+            $databaseDriver = 'Doctrine\DBAL\Driver\PDOSqlite\Driver '; // Trailing space bufferes outout assertion
```

Comment:
> Trailing space buffers outout assertion

:thinking: this sentence seems broken and I don't know how to fix it :sweat_smile: